src/autonlp/project.py

```diff
@@ -262,11 +253,59 @@ def upload(self, filepaths: List[str], split: str, col_mapping: Dict[str, str]):
             http_post(path=f"/projects/{self.proj_id}/data/add", payload=payload, token=self._token)
             logger.info(f"[{idx + 1}/{len(filepaths)}] ✅ Success!")
 
-    def train(self):
+    def train(self, noprompt=False):
         """Starts training on the models"""
+        logger.info("🔎 Running an estimate of the training cost...")
```

Comment:
suggestion:
from
🔎 Running an estimate of the training cost...
to
🔎 Calculating a cost estimate for the training...